dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -12,7 +12,8 @@ class RegistrationsControllerTest < ActionController::TestCase
       email: 'an@email.address',
       gender: 'F',
       age: '13',
-      user_type: 'student'
+      user_type: 'student',
+      email_preference_opt_in: 'yes'
```

Comment:
I'm trying to work out why this is necessary given that we're not setting `email_preference_opt_in_required: true` at the same time.